### ISSUE TITLE
Universe Sandbox x64, missing entry point at KxCrt.dll

### ISSUE THREAD

**Issue by inf0renv** [vxiiduu/VxKex#267]
Well i was trying to play Universe Sandbox and when i tried to start it, it didnt work. I used VxKex and it gave an error of a missing entry point in the file d3d11.dll. I downloaded a newer version of the file and placed it in the game directory. It solved the problem but then the game just showed another error. 
![error](https://github.com/vxiiduu/VxKex/assets/159966622/62438b45-0cf9-4379-bd68-93534c48e20a)

Here is the log file:
[Universe Sandbox x64-00133635680441163292-4620.zip](https://github.com/user-attachments/files/15939871/Universe.Sandbox.x64-00133635680441163292-4620.zip)

**Comment by CopperFr** [comment]
This fonction is not implemented there is a comment about this in wcsftime.c :

//
// A note on _Wcsftime: This function is implemented by calling _Wcsftime_l,
// which is present but non-exported in Win7 msvcrt. We can grab its location
// out of the functions wcsftime or _wcsftime_l, which are just stubs that
// call _Wcsftime_l.
//
// I will only bother doing that if there are actually apps that call it,
// though.
//

Don't know if it's possible to call the one in ucrtbase.dll like for the other function in wcsftime.c

**Comment by inf0renv** [comment]
Yeah the game is suppossed to be compatible with Windows 7. though it only is until update 33. With update 34 it started appearing this error

**Comment by CopperFr** [comment]
You're right... You can try to make it run with my newer version... But you need to remove d3d11.dll you placed in the game directory because I implemented D3D11On12CreateDevice in new kxdx11.dll  instead...
[VxKex.zip](https://github.com/user-attachments/files/16026247/VxKex.zip)

**Comment by pokycraftgamer9** [comment]
> You're right... You can try to make it run with my newer version... But you need to remove d3d11.dll you placed in the game directory because I implemented D3D11On12CreateDevice in new kxdx11.dll instead... [VxKex.zip](https://github.com/user-attachments/files/16026247/VxKex.zip)

Setup files are missing from this build

**Comment by CopperFr** [comment]
Yes because it's not possible to post exe here (and in a zip file chrome said it was dangerous)
